pkg/apis/core/v1/validation/validation.go

```diff
@@ -144,6 +158,89 @@ func ValidatePodLogOptions(opts *v1.PodLogOptions) field.ErrorList {
 	return allErrs
 }
 
+// ValidateNodeLogViewerOptions checks if options that are set are at the correct
+// value. Any incorrect value will be returned to the ErrorList.
+func ValidateNodeLogViewerOptions(opts *v1.NodeLogViewerOptions) field.ErrorList {
+	files, allErrs := validateQuery(opts.Query)
+	if files && (opts.SinceTime != nil || opts.UntilTime != nil || opts.Boot != nil || opts.TailLines != nil ||
+		len(opts.Format) > 0 || len(opts.Pattern) > 0) {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("query"), opts.Query,
+			"file query cannot be combined with options"))
+	}
+	if opts.SinceTime != nil && opts.UntilTime != nil && (opts.SinceTime.After(opts.UntilTime.Time)) {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("sinceTime", "untilTime"), "",
```

Comment:
This is not the right field path - it will show an error like `sinceTime.untilTime` - you need to pick one of the fields as the "anchor" for the error.  E.g. `untilTime` and the error says

```
 "must be after `sinceTime`"
```
 (note the back-quotes!).